static/app/utils/replays/replayerStepper.tsx

```diff
@@ -62,25 +62,25 @@ export default function replayerStepper<
     };
 
     const frameRef: FrameRef<Frame> = {
-      frame: undefined,
+      current: undefined,
     };
 
     const considerFrame = (frame: Frame) => {
       if (shouldVisitFrame(frame, replayer)) {
-        frameRef.frame = frame;
+        frameRef.current = frame;
         window.setTimeout(() => {
           const timestamp =
             'offsetMs' in frame ? frame.offsetMs : frame.timestamp - startTimestampMs;
           replayer.pause(timestamp);
         }, 0);
```

Comment:
would RAF be better here @ryan953?

Reply:
we could try it, and see if the memory chart renders the same as before. 

also, we could experiment with different granularity in the memory chart.

Reply:
Here's an answer i found on the differences: https://stackoverflow.com/questions/38709923/why-is-requestanimationframe-better-than-setinterval-or-settimeout

Reply:
seems to be the same behavior after using RAF so i pushed up the change